tests/kuttl/tests/dataplane-deploy-no-nodes-test/06-assert.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: edpm-compute-beta-nodeset
+  namespace: openstack
+spec:
+  services:
+  - download-cache
+  - bootstrap
+  env:
+  - name: ANSIBLE_FORCE_COLOR
+    value: "True"
+  nodes: {}
+  nodeTemplate:
+    ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
+status:
+  conditions:
```

Comment:
You need to add `ServiceAccountReady` condition here too.

        +  - lastTransitionTime: "2024-03-22T08:26:47Z"
        +    message: ServiceAccount created
        +    reason: Ready
        +    status: "True"
        +    type: ServiceAccountReady

Reply:
Done, config hashes too.